flexget/plugins/output/decompress.py

```diff
@@ -146,13 +138,12 @@ def is_dir(self, info):
         """
         Tests whether the file descibed in info is a directory
         """
-        if isinstance(info, zipfile.ZipInfo):
-            base = os.path.basename(info.filename)
-            return not bool(base)
-        elif isinstance(info, rarfile.RarInfo):
+
+        if hasattr(info, 'isdir'):
             return info.isdir()
         else:
-            raise ValueError('Not a RarInfo or ZipInfo object.')
+            base = os.path.basename(info.filename)
+            return not bool(base)
```

Comment:
No need to manually `bool` this, the `not` operator will do that for us. Another style note, a `try` block with `AttributeError` is probably more pythonic than `hasattr`.
https://docs.python.org/2/glossary.html#term-eafp